test/native/setup.js

```diff
@@ -20,13 +20,17 @@ jest.mock( '@wordpress/element', () => {
 	};
 } );
 
+let triggerHtmlSerialization = () => {};
+let serializedHtml;
 jest.mock( '@wordpress/react-native-bridge', () => {
 	return {
 		addEventListener: jest.fn(),
 		mediaUploadSync: jest.fn(),
 		removeEventListener: jest.fn(),
 		requestFocalPointPickerTooltipShown: jest.fn( () => true ),
-		subscribeParentGetHtml: jest.fn(),
+		subscribeParentGetHtml: jest.fn( ( callback ) => {
+			triggerHtmlSerialization = callback;
+		} ),
```

Comment:
This is a workaround to get the current HTML of the editor, similar to what we do on the UI tests to verify that the content is what we expect.

For using this in an integration test, check the helper below `getEditorHtml`.

Reply:
I decided to move this logic to the helper file, now it's located [here](https://github.com/WordPress/gutenberg/pull/32038/files#diff-d5f0be6da4449d0e7e208bc29d6ee352d7923f69d26c9c0965a478de0238bb3eR76-R80).